data/server_locations.json

```diff
@@ -4884,6 +4884,52 @@
                 "id": 6058,
                 "last_updated": "2023-09-17"
             }
+        },
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    -84.55117919999999,
+                    33.7706534
+                ]
+            },
+            "properties": {
+                "name": "Six Flags Over Georgia",
+                "active": false,
+                "area": "Georgia",
+                "address": "275 Riverside Parkway SW, Austell",
+                "status": "retired",
+                "external_url": "http://209.221.138.252/Details.aspx?location=1907",
```

Comment:
This seems to be a multimachine -- are we sure they are all retired?

Reply:
The e-mail was "Just visited over the weekend, could not find any machines throughout the park. The penny collectors website stated the machines were removed last year. " and the website also looks like it